Feeld/react-native-transparent-video
# Update specified yarn version and yarn.lock file

### Summary

This PR updates the specified `packageManager` yarn version in `package.json` to match the specified version in the `feeld-mobile` repo, of which this repo is a dependency.

This fixes an issue where users running `yarn install` in the `feeld-mobile` repo with the specified version would encounter an error when yarn tried to pack and install this repo as a dependency. EG:

<img width="852" alt="Screenshot 2023-12-13 at 10 54 06 AM" src="https://github.com/Feeld/react-native-transparent-video/assets/1670836/b40c6274-66ed-4381-819b-1c9c1549ffac">

The log itself referenced in the above screenshot looks like this:

```sh
Packing @status-im/react-native-transparent-video@https://github.com/Feeld/react-native-transparent-video.git#commit=8d80f2ffeaae37fabee3325ec7e1aae66d57d605 from sources
Using Yarn Classic for bootstrap. Reason: "__metadata" key not found in yarn.lock, must be a Yarn classic lockfile

➤ YN0000: Retrieving https://classic.yarnpkg.com/latest.js
➤ YN0000: Saving the new release in .yarn/releases/yarn-1.22.19.cjs
➤ YN0000: Done in 1s 167ms

Usage Error: Unsupported package manager specification (^yarn@1.22.15,^yarn,1.22.15)

$ yarn ...
```

#### Testing notes

After checking out the `feeld-mobile` repo locally, run `yarn install` to install your dependencies. If the version of yarn invoked is incompatible with the version specified in _this_ repo's `packageManager` specification, you should encounter the above error.

To verify that this PR resolves this issue, you can do the following in the `feeld-mobile` repo:

- Run `rm -rf node_modules` and `yarn cache clean` to reset your installed dependencies.
- Update the requested version of `@status-im/react-native-transparent-video` in `package.json` to request this branch specifically, like so:
  ```
  "@status-im/react-native-transparent-video": "github:Feeld/react-native-transparent-video#jn/update-yarn-lockfile",
  ```
- Run `yarn install` and verify that you are able to successfully install your dependencies.

There are no actual codechanges included in this PR, but if you wish, you can also run `yarn pod:install` and then build and run the app itself to verify that the transparent video dependency continues to function correctly.
